src/sklearn_evaluation/plot/roc.py

```diff
@@ -39,7 +39,14 @@ def roc(y_true, y_score, ax=None):
     .. plot:: ../examples/roc.py
 
     """
-    r = ROC(y_true, y_score, ax=ax)
+    if y_true is not None and y_score is not None:
+        warn(
+            "ROC will change its signature in version 0.10"
+            ", please use ROC.from_raw_data",
+            FutureWarning,
+            stacklevel=2,
+        )
```

Comment:
we no longer need this warning. We'll still support the functional API so there's no need for a warning and the `ROC.__init__` with pass `y_true`, `y_score` has only been out for two days so I doubt people used it.